examples/linear_static.jl

```diff
@@ -0,0 +1,104 @@
+# This file is a part of JuliaFEM.
+# License is MIT: see https://github.com/JuliaFEM/JuliaFEM.jl/blob/master/LICENSE.md
+
+# # Linear Static Example
+
+# ![](linear_static/freecad.png)
+
+# ## Preprocessing
+using JuliaFEM
+using JuliaFEM.Preprocess
+using JuliaFEM.Postprocess
+add_elements! = JuliaFEM.add_elements!
+
+# First we will read in the mesh. Geometry and mesh are greated with FreeCAD,
+# where med format is selected for exporting. Mesh file consist also edge and
+# surface mesh, which we will need to neglect later.
+
+datadir = Pkg.dir("JuliaFEM", "examples", "linear_static")
+meshfile = joinpath(datadir, "JuliaFEMSMP18.med")
+mesh = aster_read_mesh(meshfile)
+
+# Next we will create the model and define Elasticity. Also elements are added
+# to the model.
+
+model = Problem(Elasticity, "OTHER", 3)
+model_elements = create_elements(mesh, "OTHER")
+
+# Elements need material properties and they are defined next
+
+update!(model_elements, "youngs modulus", 208.0E3)
+update!(model_elements, "poissons ratio", 0.30)
+update!(model_elements, "density", 7.80E-9)
+JuliaFEM.add_elements!(model, model_elements)
+
+# Let's ingore Seg3 and Tri6 elements using `filter`
+model.elements = collect(filter(x -> !isa(x, Union{Element{Seg3},
+                                            Element{Tri6}}), model.elements));
+
+# The whole idea of the JuliaFEM input is to be a normal Julia script, where the
+# user can freely define any functions needed to perform the task. Here we
+# define a function, which finds nodes on the given plane yz, xz or xy from the
+# given height.
+
+function find_nodes_at_certain_plane!(mesh,name,vector_id,distance,radius=6.0)

```

Comment:
`add_nodes_at_certain_plane_to_node_set!` this is really good proposal. I don't like the latter because you will need to loop twice and it will look longer (more complicated).